ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs

```diff
@@ -44,6 +46,13 @@ data NodeToNodeVersion
     -- ^ Changes:
     --
     -- * Enable full duplex connections.
+    | NodeToNodeV_11
```

Comment:
I recently approved another PR which is using this version for something else.  When you will be rebasing we can use `NodeToNodeV_11` for both changes, so just merge the haddocks.

_If we go this route, then we need to make sure the next published version of `ouroboros-network-api` contains both changes._

Reply:
Got it! :)